app/models/deploy_group.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 class DeployGroup < ActiveRecord::Base
-  has_soft_deletion default_scope: true
```

Comment:
needs to stay here too in case a user does not have all plugins, see https://github.com/zendesk/samson/blob/master/app/models/stage.rb#L5